### ISSUE TITLE
Is it possible to mix those function into "Goindex" this project? :)

### ISSUE THREAD

**Issue by Sn0wl3r0ker** [FWeinb/rclone-js#6]
Is it possible to mix those function into "[Goindex](https://github.com/donwa/goindex)" this project?
cause my file upload to google drive with rclone encrypt,and i wish i can watch those encrypt videos/files on it. hope u can take a look THX : )

**Comment by FWeinb** [comment]
Yeah this should be possible. I won't have time to work on that though.

**Comment by rmacnguyen** [comment]
Probably not possible, since Cloudflare worker would exceed CPU limits.
